modules/oadp-s3-compatible-backup-storage-providers.adoc

```diff
@@ -21,13 +21,12 @@ The following AWS S3 compatible object storage providers are fully supported by
 * Ceph RADOS Gateway (Ceph Object Gateway)
 * Red Hat Container Storage
 * {odf-full}
+* Google Cloud Platform (GCP)
+* Microsoft Azure
 
 [NOTE]
 ====
-The following compatible object storage providers are supported and have their own Velero object store plugins:
-
-* Google Cloud Platform (GCP)
-* Microsoft Azure
+GCP and Microsoft Azure both have their own Velero object store plugins.
```

Comment:
```suggestion
Google Cloud Platform (GCP) and Microsoft Azure have their own Velero object store plugins.
```
I think we can drop "both" here :-) It should be clear even without it... I'm also thinking that we should define GCP here again, even though we have it defined above, because users tend to pay attention to admonitions first :-D WDYT? Feel free to ignore, of course!